src/mspdb.cpp

```diff
@@ -75,19 +79,27 @@ bool tryLoadMsPdb(const char* version, const char* mspdb, const char* path = 0)
 	return modMsPdb != 0;
 }
 
+#ifdef _M_X64
+#define MSPDB_DIR_NEW "..\\..\\VC\\bin\\amd64\\"
+#define MSPDB_DIR_OLD MSPDB_DIR_NEW
+#else
+#define MSPDB_DIR_NEW "..\\..\\VC\\bin\\"
+#define MSPDB_DIR_OLD 0
+#endif

```

Comment:
I'm not so happy with names `..._OLD` and `..._NEW`. Better use something descriptive, e.g. `BIN_DIR_LT_VS12` and `BIN_DIR_GE_VS12`.